packages/ingest-mongodb-public/src/sources/index.ts

```diff
@@ -53,9 +52,9 @@ const mongoDbUniversitySourceConstructor = async () => {
     sourceName: "mongodb-university",
     baseUrl: "https://api.learn.mongodb.com/rest/catalog",
     apiKey: universityDataApiKey,
-    tiCatalogFilterFunc: filterOnlyPublicActiveTiCatalogItems,
-    metadata: {
-      tags: ["transcript"],
+    ti: {
```

Comment:
can you rename this property to something more meaningful, like `catalogueItems`?

Reply:
I named this tiCatalogItemsConfig